src/platform/Linux/NFCCommissioningManagerImpl.cpp

```diff
@@ -593,6 +609,11 @@ CHIP_ERROR NFCCommissioningManagerImpl::SendToNfcTag(const Transport::PeerAddres
 {
     std::shared_ptr<TagInstance> targetedTagInstance = nullptr;
 
+    if (!mThreadRunning)
```

Comment:
the pattern seems to always be `if (!mThreadRunning) { StartThread(); }`

How about calling it `CHIP_ERROR EnsureProcessingThreadStarted` (no Nfc in name because that seems to stutter) and then you just call it ?